scheduler/scheduler-smartproxy-common/src/main/java/org/ow2/proactive/scheduler/smartproxy/common/AbstractSmartProxy.java

```diff
@@ -536,28 +528,25 @@ protected String getLocalUserSpace() throws NotConnectedException, PermissionExc
      * @param job
      * @param localOutputFolder  path to the output folder on local machine if null, no actions
      *                           will be performed concerning the output data for this job
-     * @param pull_url           - the url where the data is to be retrieved after the job is
+     * @param pull_urls          the urls where the data is to be retrieved after the job is
      *                           finished
      * @param newFolderName      name of the folder to be used for pushing the output
      * @param isolateTaskOutputs task output isolation (see method submit)
      * @return a String representing the updated value of the pull_url
      */
-    protected String prepareJobOutput(TaskFlowJob job, String localOutputFolder, String pull_url, String newFolderName,
-            boolean isolateTaskOutputs) throws NotConnectedException, PermissionException {
-        // if the job defines an output space
-        // and the localOutputFolder is not null
+    protected String prepareJobOutput(TaskFlowJob job, String localOutputFolder, List<String> pull_urls,
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)